Decompiler/Program.cs

```diff
@@ -535,6 +539,7 @@ private static void ParseVPK(string path)
             else
             {
                 Console.WriteLine("--- Dumping decompiled files...");
+               
 
```

Comment:
Probably want to add another option to force a DumpVPK of a specified file extension